hera_cal/redcal.py

```diff
@@ -297,37 +486,37 @@ def build_eqs(self, bls_in_data):
                     eqs['g%d%s * g%d%s_ * u%d%s' % params] = (ant_i, ant_j, pol)
         return eqs
 
-    def _solver(self, solver, data, wgts={}, detrend_phs=False, sparse=False, **kwargs):
+    def _solver(self, solver, data, wgts={}, detrend_phs=False, **kwargs):
         """Instantiates a linsolve solver for performing redcal.
 
         Args:
             solver: linsolve solver (e.g. linsolve.LogProductSolver or linsolve.LinProductSolver)
             data: visibility data in the dictionary format {(ant1,ant2,pol): np.array}
             wgts: dictionary of linear weights in the same format as data. Defaults to equal wgts.
             detrend_phs: takes out average phase, useful for logcal
-            sparse: represent the A matrix (visibilities to parameters) sparsely in linsolve
             **kwargs: other keyword arguments passed into the solver for use by linsolve
 
         Returns:
             solver: instantiated solver with redcal equations and weights
         """
-
+        # XXX ARP: concerned about detrend_phs.  Why is it necessary?
```

Comment:
I honestly don't know. Where did it come from? If I recall correctly, you put this in here.